src/main/kotlin/com/pubnub/api/enums/PNStatusCategory.kt

```diff
@@ -99,4 +99,9 @@ enum class PNStatusCategory {
      * Previously started subscribe loop did fail and at this moment client disconnected from real-time data channels.
      */
     PNConnectionError,
+
+    /**
+     * It was not possible for Event Engine to regain connection to PN server.
+     */
+    PNDisconnectedUnexpectedly,
```

Comment:
I thought that we will reuse already existing status `PNUnexpectedDisconnect`. Having both might be quite confusing

Reply:
Ok. Makes sense. I will change it.